tap_zendesk/discover.py

```diff
@@ -58,10 +58,18 @@ def discover_streams(client, config):
         streams.append({'stream': stream.name, 'tap_stream_id': stream.name, 'schema': schema, 'metadata': stream.load_metadata()})
 
     if error_list:
+
+        total_stream = len(STREAMS.values()) # Total no of streams
         streams_name = ", ".join(error_list)
-        message = "HTTP-error-code: 403, Error: You are missing the following required scopes: read. "\
-                    "The account credentials supplied do not have read access for the following stream(s):  {}".format(streams_name)
-        raise ZendeskForbiddenError(message)
+        message = "The account credentials supplied do not have read access for the following stream(s): {}."\
+            "The data for the mentioned streams will not be collected if selected due to a lack of 'read' permission.".format(streams_name)
+        if len(error_list) != total_stream:
+            # If atleast one stream have read permission then just print warning message for all streams
+            # which does not have read permission
+            LOGGER.warning(message)
+        else:
+            # If any one of stream does not have read permission then raise error.
```

Comment:
@prijendev The comment here is wrong.
This code snippet depicts that. "If none of the streams are having the 'read' access, then the code will raise an error

Reply:
Updated comment